app/routers/service-response-router.ts

```diff
@@ -14,6 +14,7 @@ export default function router(): Router {
   const result = Router();
   result.use(json({
     type: 'application/json',
+    limit: '500kb',
```

Comment:
I'm wondering what error you ran into that caused this. What happens if a callback has a body larger than 500kb - I assume the request will fail?

Since the results are mostly stored in the STAC catalog file and not sent as part of the callback I guess we shouldn't get anywhere near this limit though.

Reply:
The call to the backend was too large. I forget the name of the error but the default limit on the JSON parser body size is pretty small.

Reply:
I logged the actual size of the failed request body to get an idea of a reasonable setting.